userapp/src/components/user/UserForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import { makeStyles,
+    TextField,
+    Grid,
+}  from '@material-ui/core';
+
+  const useStyles = makeStyles((theme) => ({
+    root: {
+      '& .MuiFormControl-root': {
+        width: '80%',
+        margin: theme.spacing(1),
+      },
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+    },
+  }));
+
+/**
+ * @return {JSX} UserCreate site
+ * routed at /user/new
+ */
+const UserForm = ({ initialUser }) => {
+    const [user, setUser] = useState(initialUser);
```

Comment:
comment state variable purpose

Reply:
Commented! Overlooked it while debugging. Next time I will remember to comment it